plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -91,6 +91,8 @@ func ClusterRoles() []rbac.ClusterRole {
 			Rules: []rbac.PolicyRule{
 				// TODO add future selfsubjectrulesreview, project request APIs, project listing APIs
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("selfsubjectaccessreviews").RuleOrDie(),
+
+				rbac.NewRule(Read...).Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
```

Comment:
The power seems ok to grant and we granted it in the same place in openshift.  I'm ok with this, but if you want a separate role (still auto-bound) I'm ok with that too.